barretenberg/cpp/src/barretenberg/commitment_schemes/kzg/kzg.hpp

```diff
@@ -46,27 +46,6 @@ template <typename Curve_> class KZG {
         prover_trancript->send_to_verifier("KZG:W", quotient_commitment);
     };
```

Comment:
There isn't an actual need for the "full verificaton" function besides in kzg.test so removed to avoid duplication